docs/howto/operate/new-cluster/aws.md

```diff
@@ -1,158 +1,209 @@
 (new-cluster:aws)=
 # Add a new AWS cluster
 
-We do not currently use Terraform to deploy a new cluster in AWS.
-Instead, we use a tool [called `kops`](https://github.com/kubernetes/kops) to provision the cluster.
+We use [eksctl](https://eksctl.io/) to provision our k8s clusters on AWS.
 
-:::{admonition} We also use `eksctl`
-In addition to `kops`, we have begun using an AWS-native tool called `eksctl` to provision Kubernetes clusters on AWS.
-See [this configuration](https://github.com/2i2c-org/infrastructure/tree/master/eksctl) for an example of a hub that uses this.
-As we understand these practices better we will update this guide with more information about `eksctl`.
+:::{note}
+
+We used to use [kops](https://kops.sigs.k8s.io/) as well, for clusters
+where we wanted to minimize base running cost. We have deprecated that now, and use
+eksctl for everything.
 :::
 
-These instructions describe how to create a new AWS cluster from scratch with `kops`.
 
+## Install needed tools locally
 
-## Pre-requisites
+1. Follow the instructions outlined in [Set up and use the the deployment
+   scripts locally](operate:manual-deploy) to set up the local environment and
+   prepare `sops` to encrypt and decrypt files.
 
-1. Follow the instructions outlined in [Set up and use the the deployment scripts locally](operate:manual-deploy) to set up the local environment and prepare `sops` to encrypt and decrypt files.
+2. Install the awscli tool (you can use pip or conda to install it in the
+   environment) and configure it to use the provided AWS user credentials.  [Follow
+   this guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config)
+   for a quick configuration process.
 
-2. Install the awscli tool (you can use pip or conda to install it in the environment) and configure it to use the provided AWS user credentials.
-   [Follow this guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config) for a quick configuration process.
+3. Install the latest version of [eksctl](https://eksctl.io/introduction/#installation). Mac users
+   can get it from homebrew with `brew install eksctl`.
 
-   ```{note}
-   The customer with AWS admin privileges should have created a user for you with full privileges.
-   We will probably explore fine-graining the permissions actually needed
-   in the short-term.
+4. Put your AWS credentials (access key id and secret) in `~/.aws/credentials` file, in the following
+   format:
+
+   ```ini
+   [your-cluster-name]
+   aws_access_key_id = <key-id>
+   aws_access_secret_key = <access-key>
```

Comment:
I have successfully used AWS environment variable instead of a credentials file, but this is probably the canonical way...